test/integration_test/kubevirt_test.go

```diff
@@ -49,6 +60,13 @@ func TestKubevirt(t *testing.T) {
 	err = createDatadiskTemplates(t)
 	require.NoError(t, err, "Error creating kubevirt templates")
 
+	err = createClusterroleDataVolume(dataVolumeClusterrole)
+	require.NoError(t, err, "Error creating cluste role for datavolume clone")
+
+	err = createClusterroleBindingDatavolume(dataVolumeClusterrole, kubevirtDatadiskNamespace)
+	require.NoError(t, err, "Error creating/updating data volumeclusterrole binding %s with namespace: %s",
```

Comment:
nit: need space between volume and clusterrole

Reply:
Fixed.